src/pages/understand-stacks/network.md

```diff
@@ -100,7 +100,7 @@ Sample response for a successful call:
 }
 ```
 
--> To set the function call arguments and read the result, [Clarity values](http://localhost:3000/understand-stacks/transactions#clarity-value-types) need to be serialized into a hexadecimal string. The [Stacks Transactions JS](https://github.com/blockstack/stacks.js/tree/master/packages/transactions) library supports these operations
+-> To set the function call arguments and read the result, [Clarity values](/understand-stacks/transactions#clarity-value-types) need to be serialized into a hexadecimal string. The [Stacks Transactions JS](https://github.com/blockstack/stacks.js/tree/master/packages/transactions) library supports these operations
```

Comment:
I think we should point users to @stacks/stacking instead, there's a lot of helpers in there

Reply:
oh, this is a fix that is not related to stacking - it's on the network page. just fixing a broken URL. But yes, we do link to the stacking lib where it makes sense :)